pkg/controller/elasticsearch/certificates/transport/reconcile_test.go

```diff
@@ -7,22 +7,21 @@ package transport
 import (
 	"bytes"
 	"context"
-	"reflect"
 	"testing"
 
+	esv1 "github.com/elastic/cloud-on-k8s/v2/pkg/apis/elasticsearch/v1"
+	"github.com/elastic/cloud-on-k8s/v2/pkg/controller/common/certificates"
+	"github.com/elastic/cloud-on-k8s/v2/pkg/controller/common/comparison"
+	"github.com/elastic/cloud-on-k8s/v2/pkg/controller/elasticsearch/label"
+	"github.com/elastic/cloud-on-k8s/v2/pkg/utils/k8s"
```

Comment:
I suspect the moving of these imports wasn't intentional.

Reply:
No some autoformat thingy. Will reverse.